packages/riipen-ui/src/components/Container.test.jsx

```diff
@@ -0,0 +1,104 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import defaultTheme from "../themes/default";
+
+import Container from "./Container";
+
+describe("<Container>", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
+      mount(<Container />);
+    } catch (theError) {
+      error = theError;
+    }
+    expect(error).toEqual(undefined);
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const defaultProps = new Container().type.defaultProps;
+
+      const wrapper = mount(<Container />);
+
+      expect(wrapper.find("Container").props().maxWidth).toEqual(
+        defaultProps.maxWidth
+      );
+
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    it("appends higher order values to default classes prop with withClass decorator", () => {
+      const wrapper = mount(<Container />);
+
+      expect(wrapper.find("Container").props().classes).toEqual([
+        "riipen",
+        "riipen-container"
+      ]);
+    });
+  });
+
+  describe("children prop", () => {
+    it("displays given children", () => {
+      const child = <div>child</div>;
+
+      const wrapper = mount(<Container>{child}</Container>);
+
+      expect(wrapper.containsMatchingElement(child)).toBeTruthy();
+    });
+  });
+
+  describe("classes prop", () => {
+    it("sets custom class", () => {
+      const classes = ["newClass"];
+
+      const wrapper = mount(<Container classes={classes} />);
+
+      expect(
+        wrapper
+          .find("Container")
+          .childAt(0)
+          .hasClass(classes[0])
+      ).toEqual(true);
```

Comment:
You and @imogenem are also doing these differently. Imo is testing 2 classes and you are testing one. We should be consistent.

I think I prefer testing one rather than two as testing two doesn't add a ton of extra value.

I guess this comment is more for @imogenem on future PRs.

Reply:
@licarm let's meet?

Reply:
@jordanell Do you want previous test files adjusted to just test with one class?